### ISSUE TITLE
Remediate Connections lightbox/carousel

### ISSUE THREAD

**Issue by erinchambers** [CDRH/family_letters#201]
Before we do anything about this issue, let's discuss whether we want to remediate the existing lightbox or switch to using the same lightbox as the rest of our Orchid sites.

Family Letters uses [Lightbox for Bootstrap](https://github.com/ashleydw/lightbox) for its image lightbox. From testing this page https://cdrhrailsdev.unl.edu/family-letters/explore/connections:

- [ ] Carousel controls (left and right arrows) are read as empty links. I think we just need to add `<span class="sr-only">Previous image</span>` inside the left arrow link, and `<span class="sr-only">Next image</span>` inside the right arrow link.
- [ ] Inside the lightbox, the image doesn't have alt text, so we need to add that, as well.

**Comment by wkdewey** [comment]
It seems difficult to put alt text into the lightbox

**Comment by erinchambers** [comment]
> It seems difficult to put alt text into the lightbox

Agreed. I think the best course of action will be to use the same lightbox Orchid uses (fancybox) instead of the current Bootstrap one.

**Comment by erinchambers** [comment]
Related issue (created after 2026-08-26 meeting): #211

**Comment by techgique** [comment]
Fancybox gallery works on all the Explore pages now. I also increased the enlarged image size:
https://cdrhrailsdev.unl.edu/family-letters/en/explore

Related PR: https://github.com/CDRH/family_letters/pull/212
